packages/yourloops/test/unit/components/alarm/alarms-content-configuration.test.tsx

```diff
@@ -297,4 +273,138 @@ describe('AlarmsContentConfiguration', () => {
     expect(screen.queryByText('default-min-max')).not.toBeNull()
     expect(screen.queryAllByText('default')).toHaveLength(4)
   })
+  it('save button should be disabled when low bg value is not integer', () => {
```

Comment:
The three following unit tests are testing DOM. This should be covered by an integration test (might already be)